docker/images/toolpad/Dockerfile.demo

```diff
@@ -0,0 +1 @@
+FROM muicom/toolpad:latest
```

Comment:
Shall we pin the version

Reply:
I pinned it to v0.0.21, will just have to manually keep it updated.
Other than that does this flow with the `demo` branch look good to you if it works? Or would you still prefer we use a separate repo for the demo?